### PR TITLE
Add Unitree A1 quadruped

### PR DESCRIPTION
**Pull request by wxmerkt**
This PR adds the Unitree A1 quadruped. It's based on the latest upstream URDF (MPL-2.0 licensed) with the Gazebo and transmission tags removed.

### CODE REVIEW

File: python/example_robot_data/robots_loader.py [hunk 1]
```diff
@@ -99,6 +99,15 @@ def q0(self):
         return self.robot.q0
 
 
+class A1Loader(RobotLoader):
+    path = 'a1_description'
+    urdf_filename= "a1.urdf"
```
Comment (nim65s) [inline python/example_robot_data/robots_loader.py:104]:
```suggestion
    urdf_filename = "a1.urdf"
```

**Review by nim65s — APPROVED** [review]
thanks !

**Review by jcarpent — APPROVED** [review]
Very nice @wxmerkt.

**Comment by cmastalli** [comment]
Thanks @wxmerkt and @nim65s!

**Pull request merged**